lib/utils.js

```diff
@@ -106,4 +106,38 @@ function loadPlugin (pluginId) {
   return loadModule('plugins', pluginId)
 }
 
-module.exports = { cloneObject, toDebugString, isValidString, isInRange, isInteger, isTimestamp, loadPlugin, loadModule }
+/**
+ * Inserts content into an object by using its path in string format.
```

Comment:
Vielleicht noch wichtig: verändert den `object` parameter und gibt nichts zurück. Und: Fügt alle Subobjekte, die auf dem Weg in `path` liegen ein. Beispiel:
```javascript
let object = {}
insertByPath(object, 'a.b.c.d', 'hello')
assert object == { a: { b: { c: { d: 'hello' } } } }
```